geoips/plugins/yaml/sectors/static/abu_dhabi.yaml

```diff
@@ -0,0 +1,29 @@
+interface: sectors
```

Comment:
Thoughts on organizing the YAML sectors into subdirectories, just for clarity?  I could go either way, and do not really have thoughts on the best organization either...

Reply:
Going to wait on this until we add more sectors or have a better idea of how to organize them. Resolved.